modules/tiles/docs/README.md

```diff
@@ -65,7 +65,7 @@
   - `tileset`: tileset tile
 - `depth` (`Number`): The depth of the tile in the tileset tree.
 - `content` (`Object`): The tile's content.This represents the actual tile's payload.
-- `type` (`String`): One of `scenegraph`, `pointcloud`, `simplemesh`
+- `type` (`String`): One of `scenegraph`, `pointcloud`, `mesh`
```

Comment:
API change? specify in upgrade guide?

Reply:
this is not API change, this is added as new. one of the refactor PRs was landed to master